libs/eo/shared/atomic-design/ui-atoms/src/lib/eo-idle-timer/idle-timer-countdown.component.ts

```diff
@@ -40,13 +40,12 @@ import { Observable, filter, tap } from 'rxjs';
   `,
 })
 export class EoIdleTimerCountdownModalComponent {
+  private dialogRef = inject<WattDialogRef<EoIdleTimerCountdownModalComponent>>(WattDialogRef);
+  data = inject(WATT_DIALOG_DATA);
```

Comment:
issue: this is actually not the same as before. `data` is now type `any`, where before it was `{ countdown$: Observable<number> }`

Reply:
Put a type on